x/ccv/provider/keeper/consumer_lifecycle.go

```diff
@@ -493,9 +512,16 @@ func (k Keeper) StopConsumerChain(ctx sdk.Context, consumerId string, closeChan
 	k.DeleteAllOptedIn(ctx, consumerId)
 	k.DeleteConsumerValSet(ctx, consumerId)
 
+	k.DeleteConsumerRewardsAllocation(ctx, consumerId)
+	k.DeleteConsumerOwnerAddress(ctx, consumerId)
```

Comment:
Probably worth leaving this in as well.

Reply:
Kept the owner address. I do not see any reason to keep the consumer rewards allocation since after the chain gets deleted we won't be able to get rewards in any case.

Reply:
Owner address. The RewardsAllocation should be deleted.